CHANGELOG.md

```
@@ -1,3 +1,20 @@
+# 3.4.3
+> Published 22 April 2026
+
+### Bugfixes
```

markdownlint-cli2 (0.22.0)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)